helm/hpcc/templates/_helpers.tpl

```diff
@@ -604,6 +604,21 @@ vaults:
     {{- if index $vault "appRoleSecret" }}
       appRoleSecret: {{ index $vault "appRoleSecret" }}
     {{- end -}}
+    {{- if (hasKey $vault "retries") }}
+      retries: {{ $vault.retries }}
+    {{- end }}
+    {{- if (hasKey $vault "retry_wait_seconds") }}
+      retry_wait_seconds: {{ $vault.retry_wait_seconds }}
```

Comment:
Minor.  There seems to be a mixture of naming conventions for compound words:

verify_server
client-secret
appRoleSecret

which is confusing.  I think we have used camelCase as a standard nearly everywhere else - so these should be consistent with that.